### PR TITLE
Added cover platform to microBees Integration

### PR DESCRIPTION
**Pull request by marcolettieri**
## Proposed change

added cover platform to microBees Integration
the status of cover entity will be unknown

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [X] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/111135
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by klaasnicolaas — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged 👍

**Review by jpbede — CHANGES_REQUESTED** [review]
Please add a note as commented in core PR (https://github.com/home-assistant/core/pull/111135#discussion_r1514033176), that the cover entity will be unknown

**Review by jpbede — CHANGES_REQUESTED** [review]
There was no change to the documentation. Please see my comment above

**Review by jpbede — DISMISSED** [review]
Please do not mark a PR as "Ready for review" if it actually is not. Please include a note in the documentation that the cover entity remains in the `unknown` state, as I described in the core PR (https://github.com/home-assistant/core/pull/111135#discussion_r1514033176), otherwise we might get a lot "why is this entity unknown" issues. After addressing my comment, you can mark this as "Ready for review".

**Comment by marcolettieri** [comment]
> Please do not mark a PR as "Ready for review" if it actually is not. Please include a note in the documentation that the cover entity remains in the `unknown` state, as I described in the core PR ([home-assistant/core#111135 (comment)](https://github.com/home-assistant/core/pull/111135#discussion_r1514033176)), otherwise we might get a lot "why is this entity unknown" issues. After addressing my comment, you can mark this as "Ready for review".

we have added in note in the PR. Anyway now we will add also to comments

File: source/_integrations/microbees.markdown [hunk 1]
```diff
@@ -1,7 +1,8 @@
 ---
 title: microBees
 description: Instructions on how to integrate microBees devices into Home Assistant.
-ha_category:
+ha_category:  - Switch
```
Comment (c0ffeeca7) [inline source/_integrations/microbees.markdown]:
```suggestion
ha_category: 
```

File: source/_integrations/microbees.markdown [hunk 2]
```diff
@@ -1,7 +1,8 @@
...
+  - Cover
```
Comment (c0ffeeca7) [inline source/_integrations/microbees.markdown:5]:
```suggestion
  - Switch
  - Cover
```

File: source/_integrations/microbees.markdown (hunk 1 above)
Comment (c0ffeeca7) [reply source/_integrations/microbees.markdown]: Tweak to fix the build

**Comment by c0ffeeca7** [comment]
@marcolettieri: Please implement the comment by jpbede. I see you added a note in the PR description. But this is not visible in the documentation. You need to add this information to the file itself.

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @marcolettieri :+1:

**Comment by c0ffeeca7** [comment]
@jpbede, are you good with the PR?

**Comment by jpbede** [comment]
Yes, this is fine now, thanks @marcolettieri 👍

**Pull request merged**